app/src/lang/translations/en-US.yaml

```diff
@@ -289,19 +289,12 @@ foreign_key: Foreign Key
 finish_setup: Finish Setup
 dismiss: Dismiss
 raw_value: Raw value
-copy_raw_value: Copy Raw Value
-copy_raw_value_success: Copied
-copy_raw_value_fail: Copy Failed
-paste_raw_value: Paste Raw Value
-paste_raw_value_success: Pasted
-paste_raw_value_fail: Paste Failed
-view_raw_value: View Raw Value
```

Comment:
@rijkvanzanten something seems odd here. These keys were added in #10992, but this merge removed them instead. Additionally, `clear_value` + `reset_to_default` + `undo_change` were reverted to their previous value in en-US, and removed from all other locales 🤔

Reply:
Just went here to say the same thing. I already thought I have my local repo blown up, but in the end it seems this was the cause 😕